packages/canon/src/components/Input/Input.tsx

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright 2024 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use client';
+
+import React, { forwardRef } from 'react';
+import clsx from 'clsx';
+import { Field } from '@base-ui-components/react/field';
+import type { InputProps } from './types';
+
+/** @public */
+const Input = forwardRef<HTMLDivElement, InputProps>((props, ref) => {
+  const {
+    className,
+    label,
+    description,
+    errorMatch = 'customError',
+    errorForceShow = false,
+    errorMessage = 'An error occurred',
+    validationMode = 'onChange',
+    size = 'md',
+    placeholder,
+    required = false,
```

Comment:
Hmm, feeling we might need a bit more of a pattern for how we forward props if that's not provided by Base UI. Perhaps some of the more specific things like `errorForceShow` should be forwarded with a full `errorProps` or something like that instead? Or even a more general `props: { error: { ... } }` kind of pattern.

Reply:
We now split Field and Input so this solved this issue.